web_dialog_size/static/src/js/web_dialog_size.js

```diff
@@ -1,34 +1,41 @@
 odoo.define('web_dialog_size.web_dialog_size', function (require) {
 'use strict';
 
-var Model = require('web.DataModel');
+var rpc = require('web.rpc');
 var Dialog = require('web.Dialog');
 
-Dialog.include({
+var DialogSize = Dialog.include({
 
-    init: function (parent, options) {
+    willStart: function () {
         var self = this;
-        this._super.apply(this, arguments);
-        self.$modal.find('.dialog_button_extend').on('click', self.proxy('_extending'));
-        self.$modal.find('.dialog_button_restore').on('click', self.proxy('_restore'));
-
-        new Model('ir.config_parameter').query(['key', 'value']).
```

Comment:
Why did you removed the call to ir.config.parameter ?

I see the new implementation more complex, with a new model web.dialog.size.backend that only return a parameter... Is it a new norm ?

Reply:
Model class doesn't exist in v11. We should user 'web.rpc' instead and I thought it was a easiest to do this.

Reply:
I mean, I don't understand why you call a new created python class.
something like that will avoid to include web.dialog.size.backend. don't you think ?

```
rpc.query({
    mode: 'ir.config_parameter',
    method: 'search',
})
```